cfme/networks/topology_view.py

```diff
@@ -0,0 +1,58 @@
+from widgetastic_patternfly import FlashMessages, Button, Input
+from widgetastic.widget import View, Text, Checkbox
+from widgetastic.log import logged
+
+from widgetastic_manageiq import BaseEntitiesView
+from cfme.base.login import BaseLoggedInPage
+
+
+class TopologySearch(View):
+    """ Represents search_text control of TopologyView """
+    search_text = Input(id="search")
+    search_btn = Text('//*[@id="main-content"]/div[2]/div/div[1]/div/div/form/div[2]/button')
```

Comment:
Is it possible to use `widgetastic_patternfly.Button`?

Reply:
No, because those widgets has any of the required initializing values of Button widget (yet).